src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -254,6 +256,12 @@ namespace winrt::TerminalApp::implementation
         return RS_(L"ApplicationVersionUnknown");
     }
 
+    winrt::hstring TerminalPage::ThirdPartyNoticesLink()
+    {
+        winrt::hstring link{ fmt::format(std::wstring_view{ L"https://github.com/microsoft/terminal/blob/{}/NOTICE.md" }, CurrentCommitHash) };
```

Comment:
Does this first param have to be a `wstring_view`? Could it just be a string literal?

Reply:
You are definitely right about that, it should just be a string literal